lib/utils/multi-platforms.js

```diff
@@ -0,0 +1,24 @@
+var WWW_FOLDER = {
+    android: 'platforms/android/assets/www',
+    ios: 'platforms/ios/www',
+    windows: 'platforms/windows/www'
```

Comment:
I believe `windows` is not supported so you should really remove this - correct?